src/map/services/map-services.config.ts

```diff
@@ -283,65 +283,78 @@ export const getLocationDefinitionListBlock = (result: any, gridArea: string): D
   }
 }
 
-const getShowInTableBlock = (filters: {
-  key: string
-  value?: string | null
-}): DetailResultItemLinkList[] => [
-  {
-    type: DetailResultItemType.LinkList,
-    title: GLOSSARY.DEFINITIONS.NUMMERAANDUIDING.plural,
-    infoBox: getInfoBox({
-      description: GLOSSARY.DEFINITIONS.NUMMERAANDUIDING.description,
-      url: GLOSSARY.DEFINITIONS.NUMMERAANDUIDING.url,
-      plural: GLOSSARY.DEFINITIONS.NUMMERAANDUIDING.plural,
-    }),
-    links: [
-      {
-        to: {
-          pathname: config[DataSelectionType.BAG].path,
-          search: `${PARAMETERS.VIEW}=volledig&${PARAMETERS.FILTERS}={"${filters.key}":"${filters.value}"}`,
+const getShowInTableBlock = (
+  filters: {
+    key: string
+    value?: string | null
+  }[],
+): DetailResultItemLinkList[] => {
+  const filterParamValue = JSON.stringify(
+    filters.reduce(
+      (acc, { key, value }) => ({
+        ...acc,
+        [key]: value,
+      }),
+      {},
+    ),
+  )
```

Comment:
Might be cleaner to use [Object.fromEntries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries) here (untested):
```suggestion
  const filterParamValue = JSON.stringify(
    Object.fromEntries(filters.map(({ key, value }) => [key, value])),
  )
```